fetch.js

```diff
@@ -1,21 +1,31 @@
 const q = require(`sql-concat`)
-const camelize = require(`camelize`)
+const camelcaseKeys = require(`camelcase-keys`)
+
+const columns = [
+	`COLUMN_NAME`,
+	`DATA_TYPE`,
+	`COLUMN_TYPE`,
+	`NUMERIC_PRECISION`,
+	`NUMERIC_SCALE`,
+	`CHARACTER_MAXIMUM_LENGTH`,
+	`IS_NULLABLE`,
+]
 
 module.exports = (db, { schema, table }) => {
 	return new Promise((resolve, reject) => {
-		let query = q.select(`column_name`, `data_type`, `column_type`, `numeric_precision`, `numeric_scale`, `character_maximum_length`, `is_nullable`)
-			.from(`information_schema.columns`)
-			.where(`table_schema`, schema)
+		let query = q.select(columns)
```

Comment:
I think this needs to be `...columns`?  q.select expects any number of string arguments, not an array